docs/general/polkadot-vault.md

```diff
@@ -0,0 +1,135 @@
+|                                 Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/).                                  |                                                                                                                                                                                                                                                                            If there is no wired connection, side-channel attacks are not possible.                                                                                                                                                                                                                                                                             |
+|                                                                              One mnemonic seed phrase for all intra- and inter-currency accounts.                                                                              | One mnemonic for each account or one mnemonic for multiple accounts via [account derivation](../learn/learn-account-advanced.md#derivation-paths) or default [Substrate address format](../learn/learn-account-advanced.md#address-format). Although the general recommendation is to use different accounts on different chains, the final choice depends on the user's needs. See [this support article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-#Should-I-use-the-same-account-across-different-chains?) for more information. |
+|                                                                          Easy firmware and application upgrades through the Ledger Live application.                                                                           |                                                                                                                                                    No need to update the app as the same version will work fine forever and will not stop working if you do not update it. For [app upgrades](#update-the-vault-app), users must factory reset the phone and recover all accounts through seed phrases. Metadata updates for each chain must be done via QR code fountain.                                                                                                                                                     |
```

Comment:
Removing the "works forever" part and mentioning why the users may need a factory reset.

```suggestion
|                                                                          Easy firmware and application upgrades through the Ledger Live application.                                                                           |                                                                                                                                                    The app should never be connected to the internet after installation, so the version installed on the phone should not be updated directly. For [app upgrades](#update-the-vault-app), users must factory reset the phone and recover all accounts through seed phrases. Metadata updates for each chain must be done via the QR code fountain.                                                                                                                                                     |
```